tests/Tests.elm

```diff
@@ -27,6 +26,28 @@ exampleAsString =
 """
         |> String.trim
 
+selfClosingExampleAsString : String
+selfClosingExampleAsString =
+  """
+<person>
+    <name is="me">kitofr</name>
+    <here is="false" />
+    <there is="true" />
```

Comment:
What about when there are two tags, self-closing, one after the other? e.g `<there/><there/>`?